be/src/Project/Services/Identity/Identity.Api/Areas/Home/Repositories/Content/IContentRepository.cs

```diff
@@ -1,9 +1,11 @@
-﻿using System.Threading.Tasks;
+﻿using Identity.Api.Areas.Home.DomainModels;
+using System.Threading.Tasks;
 
 namespace Identity.Api.Areas.Home.Repositories.Content
 {
     public interface IContentRepository
     {
         Task<DomainModels.Content> GetContentAsync(string contentPageKey, string language);
+        Task<Policy> GetPolicyAsync(string language);
```

Comment:
**suggestion:** let's move it to a dedicated IPolicyRepository and PolicyRepository